cap/modules/schemas/views.py

```diff
@@ -49,6 +53,26 @@
 _admin_permission = admin_permission_factory(None)
 
 
+def get_all_versions(name=None):
+    """Get all versions of a schema that user has access to."""
+    if name:
+        try:
+            schemas = Schema.get_all_versions(name)
+            latest = schemas.first()
```

Comment:
it is not the correct query: `first()` will return what?..will they be sorted

Reply:
It will return the latest version because `schemas` is a sqlalchemy object and we are doing this for schemas:

```
cls.query.filter_by(name=name) \
               .order_by(cls.major.desc(),
                      cls.minor.desc(),
                      cls.patch.desc())
```